FRCore/FRCore/Keychain/SecuredKey.swift

```diff
@@ -161,15 +161,15 @@ public struct SecuredKey {
     
     /// Encrypts Data object using SecuredKey object
     /// - Parameter data: Encrypted Data object
-    public func encrypt(data: Data) -> Data? {
+    public func encrypt(data: Data, secAlgorithm: SecKeyAlgorithm = .eciesEncryptionCofactorVariableIVX963SHA256AESGCM) -> Data? {
         
-        guard SecKeyIsAlgorithmSupported(publicKey, .encrypt, algorithm) else {
-            Log.e("\(algorithm) is not supported on the device.")
+        guard SecKeyIsAlgorithmSupported(publicKey, .encrypt, secAlgorithm) else {
+            Log.e("\(oldAlgorithm) is not supported on the device.")
```

Comment:
can you confirm is this suppose to be   - Log.e("\(secAlgorithm) is not supported on the device.") ?

Reply:
good catch